Cargo.toml

```diff
@@ -9,14 +9,19 @@ lto = true          # Enable Link Time Optimization
 codegen-units = 1   # Reduce number of codegen units to increase optimizations.
 panic = 'abort'     # Abort on panic
 
+[features]
+default = ["daemon"]
+daemon = ["actix-web", "actix-rt", "futures", "read-url", "env_logger"]
```

Comment:
do you also want to remove `env_logger` ?
it shoudn't add a big increase in compile time, and logging can be helpfull even when validating local files no? it is never use?

Reply:
hm, i removed it since it doesn't seem to say much when not using daemon mode — I mostly never see anything except the initial `Starting validation` trace (at a glance, `log::*` seems to be used only two times outside of the daemon (checked with ripgrep) — or have i overlooked something?)

but if you prefer i can add it back in — as you say, it does not add much.

Reply:
oh, sorry missed it in the daemon, fine for me